include/arch/x86/linker.ld

```diff
@@ -137,6 +137,19 @@ SECTIONS
 
 #include <linker/common-ram.ld>
 
+#ifdef CONFIG_MMU
+       /*
+        * Page Tables are located here if MMU is enabled.
+	*/
+	SECTION_DATA_PROLOGUE(mmu_page_tables, (OPTIONAL),)
```

Comment:
I thought it would make it more understandable/clean than placing the entire thing in .data section. It might look a bit cluttered at the end.